docs/scos/dev/example-modules.md

```diff
@@ -0,0 +1,59 @@
+
+## The difference between core and example modules
```

Comment:
Please also ask legal's opinion on this document, as it is very legal content with promises to customers.

Reply:
This is WIP now, legal answered but needs some time to finish this.

Reply:
The test was reviewed by legal and I've got this as answer:
> Thanks for the feedback. Sounds/all looks good, thanks.